### PR TITLE
Migrate to algomate

### PR DESCRIPTION
**Pull request by alexstaeding**
Use simpler and less verbose `algomate` plugin for buildscript configuration

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,55 +1,26 @@
...
+    studentId = null
+    firstName = null
+    lastName = null
 
-application {
-    mainClass.set("h00.Main")
+    // Optionally require own tests for mainBuildSubmission task. Default is false
+    requireTests = false
+    // Optionally require public grader for mainBuildSubmission task. Default is false
+    requireGraderPublic = false
```
Comment (Rdeisenroth) [inline build.gradle.kts]: Do we need this line? We will not have a public grader till exercise 09

**Review by Rdeisenroth — CHANGES_REQUESTED** [review]
please remove references to public grader.

File: build.gradle.kts (hunk 1 above)
Comment (alexstaeding) [reply build.gradle.kts]: Fixed

**Review by Rdeisenroth — APPROVED** [review]
LGTM

**Pull request merged**